wasmedge-ggml-llama-interactive/README.md

```diff
@@ -17,9 +17,32 @@ source $HOME/.zshenv
 
 ### For Ubuntu (>= 20.04)
 
-Because we enabled OpenBLAS on Ubuntu, you must install `libopenblas-dev` by `apt update && apt install -y libopenblas-dev`.
+#### CUDA enabled
 
-Install WasmEdge 0.13.4+WASI-NN ggml plugin(OpenBLAS enabled) via installer
+The installer from WasmEdge 0.13.5 will detect cuda automatically.
+
+If it will try to install the cuda enabled version of plugin instead once the cuda is detected.
```

Comment:
If CUDA is detected, the installer will always attempt to install a CUDA-enabled version of the plugin.

Reply:
Updated